source/doc_versions.py

```diff
@@ -1,9 +1,9 @@
 # The API version strings to display in the documentation
 # COMMON_VERSION is the latest version of any one API, and is considered the overall API version
-COMMON_VERSION = '2016-06-12'
+COMMON_VERSION = '2016-07-01'
```

Comment:
It's not an internal only use case though.  The changes being made to the API are public facing, just only in the structure of the error responses that come back.  We don't record the error structure on the API doc though so there are no changes made to the documentation.